src/UI/DataHTMLTooltip.js

```diff
@@ -62,8 +62,10 @@ ${css}
         this.autofillButtons.forEach((btn) => {
             this.registerClickableButton(btn, () => {
                 // Fire only if the cursor is hovering the button
-                if (btn.matches('.tooltip__button:hover, .currentFocus')) {
+                if (btn.matches('.wrapper:not(.top-autofill) button:hover, .currentFocus')) {
```

Comment:
In these we check that the button does have the hover state, either the browser's native or our own `.currentFocus`.